common/config/.golangci.yml

```diff
@@ -29,75 +29,6 @@ linters:
           deny:
             - pkg: github.com/gogo/protobuf
               desc: gogo/protobuf is deprecated, use golang/protobuf
-        AllGoFiles:
```

Comment:
We shouldn't be modifying anything in common/ from the istio repo to the master branch. The changes need to be introduced to the common-files repo.

Reply:
Yeah ideally we wouldn't need this reordering

Reply:
I think the diff is due to https://github.com/istio/istio/pull/55828. 

@aslakknutsen any idea why the rebase took this and not the newer file? any change we did there in particular?

Reply:
@dhawton in the meantime I reverted this change in https://github.com/istio/istio/pull/56866 - can you review?